doc/ConnectionPoolingDeprecated.md

```diff
@@ -0,0 +1,60 @@
+## Using Connection Pools
+
+### Single Connection Pool (DEPRECATED)
+
+DEPRECATED VERSION
+
+Instead of creating a connection each time your client application needs to access Snowflake, you can define a cache of Snowflake connections that can be reused as needed.
+Connection pooling usually reduces the lag time to make a connection. However, it can slow down client failover to an alternative DNS when a DNS problem occurs.
+
+The Snowflake .NET driver provides the following functions for managing connection pools.
+
+| Function                                        | Description                                                                                             |
+|-------------------------------------------------|---------------------------------------------------------------------------------------------------------|
+| SnowflakeDbConnectionPool.ClearAllPools()       | Removes all connections from the connection pool.                                                       |
+| SnowflakeDbConnection.SetMaxPoolSize(n)         | Sets the maximum number of connections for the connection pool, where _n_ is the number of connections. |
+| SnowflakeDBConnection.SetTimeout(n)             | Sets the number of seconds to keep an unresponsive connection in the connection pool.                   |
+| SnowflakeDbConnectionPool.GetCurrentPoolSize()  | Returns the number of connections currently in the connection pool.                                     |
+| SnowflakeDbConnectionPool.SetPooling()          | Determines whether to enable (`true`) or disable (`false`) connection pooling. Default: `true`.         |
+
```

Comment:
I would add a note somewhere that for old pool some newly created configurations doesn't work.
For example:
- poolingEnabled - has no effect
- ChangedSession - has no effect, default is OriginalPool
- minPoolSize - has no effect, feature not available
- WaitingForIdleSessionTimeout - has no effect, feature not available